src/Runtime/Core/Collections/EmpiricalProbabilityCalibrationPoint.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Probabilistic.Collections
+{
+    using Microsoft.ML.Probabilistic.Utilities;
+
+    public struct EmpiricalProbabilityCalibrationPoint
```

Comment:
This is an awkward name.  How about `EmpiricalAndPredictedProbability` or simply `Calibration`?

Reply:
The original name I thought of was CalibrationPoint, however I was concerned that the intent of the class might not be clear, I'll go with EmpiricalAndPredictedProbability.

Reply:
How about `CalibrationPair`?  That's pretty clear.